sourceaddrs/source_registry_final.go

```diff
@@ -24,9 +27,39 @@ type RegistrySourceFinal struct {
 // string.
 var _ FinalSource = RegistrySourceFinal{}
 
+func looksLikeFinalRegistrySource(given string) bool {
+	var pkg, path string
+	if matches := finalRegistrySourcePattern.FindStringSubmatch(given); len(matches) != 0 {
+		pkg = matches[1]
+		path = matches[3]
+	}
+	return looksLikeRegistrySource(fmt.Sprintf("%s//%s", pkg, path))
+}
+
 // finalSourceSigil implements FinalSource
 func (s RegistrySourceFinal) finalSourceSigil() {}
 
+// ParseRemoteSource parses the given string as a final registry source
```

Comment:
The name in the comment doesn't match the name of the function. 😀

Reply:
Whoops! Fixed, thanks.